test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -444,15 +447,25 @@
         (is (thrown-with-msg?
               IllegalArgumentException
               #"Invalid script string in webserver 'post-config-script' configuration"
-              (jetty9/start-webserver!
-                (jetty9/initialize-context)
-                (merge base-config
-                       {:post-config-script (str "AHAHHHGHAHAHAHEASD!  OMG!")})))))
+              (let [context (jetty9/initialize-context)]
+                (with-test-logging
+                 (try
+                   (jetty9/start-webserver!
```

Comment:
Maybe we should look into creating a `with-webserver-running` testutils macro to cut down on the need to have a try/finally and shutdown call everywhere these get used?  Was thinking we might have done something like that already at some point but maybe not?  Just a suggestion - wouldn't hold up the PR for this...